components/LinkBack/LinkBack.styled.js

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+import { StyledLink } from "../../components/StyledLink/StyledLink.styled.js";
+
+export const StyledLinkBack = styled(StyledLink)`
```

Comment:
Why did you styled your StyledLink instead of putting these propertys directly to your StyledLink?

Reply:
im using `StyledLink` to actually style my links for now.
`StyledLinkBack` + `StyledLinkTo` im using for positioning and giving different props :)